src/Controller/CRUDController.php

```diff
@@ -1658,23 +1671,27 @@ protected function handleXmlHttpRequestSuccessResponse(Request $request, object
         ], Response::HTTP_OK);
     }
 
-    final protected function assertObjectExists(Request $request): void
+    final protected function assertObjectExists(Request $request, bool $strict = false): void
```

Comment:
In which cases you want to call it with false?

Reply:
For listAction and createAction.

There is no id in the request for the current admin.
But if it's a childAdmin, there will be id for the parent admin, grand-parent admin, and so on.